handlers/ph_logger.go

```diff
@@ -104,59 +107,64 @@ func (h *PHLogger) ackDelivery(d amqp.Delivery) {
 	}
 }
 
-func (h *PHLogger) RFC5424Worker(deliveries <-chan amqp.Delivery) error {
+func (h *PHLogger) deliveryToResource(d amqp.Delivery) (*logging.Resource, error) {
+	syslogMessage, err := h.parser.Parse(d.Body)
+	if err != nil {
+		return nil, err
+	}
+	if syslogMessage == nil || syslogMessage.Message() == nil {
+		return nil, errNoMessage
+	}
+	resource, err := h.processMessage(syslogMessage)
+	if err != nil {
+		return nil, err
+	}
+	return resource, nil
+}
+
+func (h *PHLogger) flushBatch(buf *[]logging.Resource, count int) {
+	fmt.Printf("Batch flushing %d messages\n", count)
+	_, err := h.client.StoreResources(*buf, count)
+	if err != nil {
+		fmt.Printf("Batch sending failed: %v\n", err)
```

Comment:
Probably should log using the logrus method to provide contextual logging instead of an ad-hoc log?